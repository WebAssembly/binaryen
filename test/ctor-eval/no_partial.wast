(module
  (memory 256 256)
  (data (i32.const 10) "waka waka waka waka waka")
  (export "test1" $test1)
  (func $test1
    (i32.store8 (i32.const 12) (i32.const 115)) ;; a safe store, should alter memory
    (unreachable)
    (i32.store8 (i32.const 13) (i32.const 114)) ;; a safe store, should alter memory, but we trapped already, and so must roll back the first one too
  )
)
