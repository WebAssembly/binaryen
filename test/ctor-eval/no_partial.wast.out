(module
 (type $0 (func))
 (memory $0 256 256)
 (data (i32.const 0) "\00\00\00\00\00\00\00\00\00\00waka waka waka waka waka")
 (export "test1" (func $test1))
 (func $test1 (type $0)
  (i32.store8
   (i32.const 12)
   (i32.const 115)
  )
  (unreachable)
  (i32.store8
   (i32.const 13)
   (i32.const 114)
  )
 )
)
